ellie/monopoly.py

```diff
@@ -0,0 +1,154 @@
+        elif self.houses == 4:
+            owner.money += self.rent4
+            renter.money -= self.rent4
+            print(f"You landed on {self.name}. Rent with {self.houses} houses costs ${self.rent4}.")
+            print(f"{renter.name} paid {owner.name} ${self.rent4}.")
+        elif self.houses >= 5:
+            owner.money += self.rent5
+            renter.money -= self.rent5
+            print(f"You landed on {self.name}. Rent with 1 hotel costs ${self.rent5}.")
+            print(f"{renter.name} paid {owner.name} ${self.rent5}.")
+
+mediterranean_ave = Property("Mediterranean Avenue", 60, 50, 2, 10, 30, 90, 160, 250, 30)
+baltic_ave = Property("Baltic Avenue", 60, 50, 4, 20, 60, 180, 320, 450, 30)
+oriental_ave = Property("Oriental Avenue", 100, 50, 6, 30, 90, 270, 400, 550, 50)
+vermont_ave = Property("Vermont Avenue", 100, 50, 6, 30, 90, 270, 400, 550, 50)
+connecticut_ave = Property("Connecticut Avenue", 120, 50, 8, 40, 100, 300, 450, 600, 60)
+st_charles_place = Property("St. Charles Place", 140, 100, 10, 50, 150, 450, 625, 750, 70)
+states_ave = Property("States Avenue", 140, 100, 10, 50, 150, 450, 625, 750, 70)
+virginia_ave = Property("Virginia Avenue", 160, 100, 12, 60, 180, 500, 700, 900, 80)
+st_james_place = Property("St. James Place", 180, 100, 14, 70, 200, 550, 750, 950, 90)
+tennessee_ave = Property("Tennessee Avenue", 180, 100, 14, 70, 200, 550, 750, 950, 90)
+new_york_ave = Property("New York Avenue", 200, 100, 16, 80, 220, 600, 800, 1000, 100)
+kentucky_ave = Property("Kentucky Avenue", 220, 150, 18, 90, 250, 700, 875, 1050, 110)
+indiana_ave = Property("Indiana Avenue", 220, 150, 18, 90, 250, 700, 875, 1050, 110)
+illinois_ave = Property("Illinois Avenue", 240, 150, 20, 100, 300, 750, 925, 1100, 120)
+atlantic_ave = Property("Atlantic Avenue", 260, 150, 22, 110, 330, 800, 975, 1150, 130)
+ventnor_ave = Property("Ventnor Avenue", 260, 150, 22, 110, 330, 800, 975, 1150, 130)
+marvin_gardens = Property("Marvin Gardens", 280, 150, 24, 120, 360, 850, 1025, 1200, 140)
+pacific_ave = Property("Pacific Avenue", 300, 200, 26, 130, 390, 900, 1100, 1275, 150)
+north_carolina_ave = Property("North Carolina Avenue", 300, 200, 26, 130, 390, 900, 1100, 1275, 150)
+pennsylvania_ave = Property("Pennsylvania Avenue", 320, 200, 28, 150, 450, 1000, 1200, 1400, 160)
+park_place = Property("Park Place", 350, 200, 35, 175, 500, 1100, 1300, 1500, 175)
+boardwalk = Property("Boardwalk", 400, 200, 50, 200, 600, 1400, 1700, 2000, 200)
+
+brown_group = [mediterranean_ave, baltic_ave]
+light_blue_group = [oriental_ave, vermont_ave, connecticut_ave]
+pink_group = [st_charles_place, states_ave, virginia_ave]
+orange_group = [st_james_place, tennessee_ave, new_york_ave]
+red_group = [kentucky_ave, indiana_ave, illinois_ave]
+yellow_group = [atlantic_ave, ventnor_ave, marvin_gardens]
+green_group = [pacific_ave, north_carolina_ave, pennsylvania_ave]
+dark_blue_group = [park_place, boardwalk]
```

Comment:
This is actually going to end up being a little tricky to design: How to represent groups? Should each property have a `group` attribute so it knows what group it's in? But then how will you find all the other properties in a group?

Again, no objectively right answer, just a fun question that becomes more obvious as you start working with classes!